packages/scripts/build-package.ts

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node --experimental-specifier-resolution=node --loader=tsx
+
+import { rm, cp } from "node:fs/promises";
+import { join } from "node:path";
+import { totalist } from "totalist";
+import { build } from "esbuild";
+
+const args = process.argv.slice(2);
+const watch = args.includes("--watch");
+
+const entryPoints: string[] = [];
+const assets: string[] = [];
+
+await totalist("./src", (rel) => {
+  if (rel.endsWith(".ts") || rel.endsWith(".tsx")) {
+    if (rel.includes(".test.") || rel.includes(".stories.")) {
+      return;
+    }
+    entryPoints.push(join("src", rel));
```

Comment:
out of curiosity - why not flat-bundling? to cover for `/server`?

Reply:
Bundling breaks code splitting. All bundlers do it based on modules.